Cargo.toml

```diff
@@ -9,7 +9,7 @@ members = [
 
 [workspace.package]
 edition = "2021"
-rust-version = "1.75"
+rust-version = "1.77"
```

Comment:
Bumped because [`split_first_chunk` was stabilized on `1.77`](https://doc.rust-lang.org/std/primitive.slice.html#method.split_first_chunk). That API used together with [`u32::from_le_bytes`](https://doc.rust-lang.org/std/primitive.u32.html#method.from_le_bytes) to encode the `NoteScript` to be used with the advice_map.